airflow/cli/utils.py

```diff
@@ -40,3 +40,10 @@ def is_stdout(fileio: IOBase) -> bool:
     .. warning:: *fileio* must be open for this check to be successful.
     """
     return fileio.fileno() == sys.stdout.fileno()
+
+
+def print_export_output(command_type: str, exported_items: Collection, file_is_stdout: bool, file_path: str):
```

Comment:
It would be better to simply pass in the file object as an argument and calculate both values here.

Reply:
@uranusjr Updated the code to reflect the suggestions, please check.